fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/F-PLN/MfdFmsFplnArr.tsx

```diff
@@ -79,6 +91,10 @@ export class MfdFmsFplnArr extends FmsPage<MfdFmsFplnArrProps> {
 
   private apprButtonScrollTo = Subject.create<number>(0);
 
+  private unit = Subject.create<Unit<UnitFamily.Distance>>(
+    NXDataStore.get('CONFIG_USING_METRIC_UNIT') === '1' ? UnitType.METER : UnitType.FOOT,
+  );
```

Comment:
We immediately get the value after render below, so to save having to duplicate the logic and default value, just set metre initially, and also a slightly better name.
```suggestion
  private lengthUnit = Subject.create<Unit<UnitFamily.Distance>>(UnitType.METER);
```